applications/yc_w16.md

```diff
@@ -0,0 +1,434 @@
+# YC Application
+
+hackEDU's nonprofit YC application for W16.
```

Comment:
In Australia we are taught to not use contractions for anything formal... But then as I have learnt recently, Australian English varies from American English in matters like this!